portal-impl/src/com/liferay/portal/service/permission/UserPermissionImpl.java

```diff
@@ -100,6 +100,10 @@ public boolean contains(
 			if (userId != ResourceConstants.PRIMKEY_DNE) {
 				user = UserLocalServiceUtil.getUserById(userId);
 
+				if (PortalUtil.isOmniadmin(permissionChecker.getUser())) {
```

Comment:
In this case, you can call `permissionChecker.isOmniadmin()` directly.

Reply:
Will do, thanks Drew!